### PR TITLE
Fix props access in transform-geojson

### PR DESCRIPTION
**Pull request by gnestor**
Simple fix 👍

### CODE REVIEW

File: packages/transform-geojson/src/index.js [hunk 1]
```diff
@@ -89,17 +89,18 @@ export class GeoJSONTransform extends React.Component<Props> {
   getTileLayer = (): ?TileLayer => {
     if (!this.el) return;
     const theme = getTheme(this.props.theme, this.el);
-    // const urlTemplate = (metadata && metadata.url_template) ||
+    // const urlTemplate = (this.props.metadata && this.props.metadata.url_template) ||
     //   'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
     const urlTemplate =
-      this.props.metadata.url_template ||
+      (this.props.metadata && this.props.metadata.url_template) ||
```
Comment (rgbkrk) [inline packages/transform-geojson/src/index.js:95]: nice!

**Comment by rgbkrk** [comment]
This fail to `build:apps` is unrelated. I'll go ahead and bring this in + the vega fix PR.

**Comment by gnestor** [comment]
Is it easy to publish a new release for just @nteract/transform-geojson?

**Comment by rgbkrk** [comment]
I'm planning on making a release tonight or tomorrow anyways, so you'll have access to it :soon:.

**Pull request merged**